UPGRADE.md

```diff
@@ -77,8 +85,30 @@ There is a list of all the repositories maintained by monorepo, changes in log b
         - `shopt -u dotglob`
         - `exit`
     - Start containers `docker-compose up -d`
-- configuration files (`config.yml`, `config_dev.yml`, `config_test.yml`, `security.yml` and `wysiwyg.yml`) has been split into packages config files (see `app/config/packages` folder)
-    - update appropriate package configs in case you had some custom configuration set in those files
+- configuration files (`config.yml`, `config_dev.yml`, `config_test.yml`, `security.yml` and `wysiwyg.yml`) has been split into packages config files, for details [see #449](https://github.com/shopsys/shopsys/pull/449)
+    - extract each section into own config file
+        - eg. from `config.yml` extract `doctrine:` section into file `packages/doctrine.yml`
+        - eg. from `config_dev.yml` extract `assetic:` section info file `packages/dev/assetic.yml`
+        - and also split `wysiwyg.yml` into `packages/*.yml`
+            - *(since `config.yml` will include all files in `packages/`, splitted `wysiwyg.yml` will be included automatically)*
+    - move `security.yml` to `packages/security.yml`
+    - the only think that have to left in original configuration files is import of these new configuration files
+        - eg. `config_dev.yml` will contain only
+            ```
+            imports:
+                 - { resource: packages/dev/*.yml }
+            ```
+- phing targets and console commands for working with elasticsearch were renamed, so rename them in `build.xml`, `build-dev.xml`. Also if you call them from other places, rename calling too:
+    - phing targets:
+        - `elasticsearch-indexes-create` -> `microservice-product-search-create-structure`
+        - `elasticsearch-indexes-delete` -> `microservice-product-search-delete-structure`
+        - `elasticsearch-indexes-recreate` -> `microservice-product-search-recreate-structure`
+        - `elasticsearch-products-export` -> `microservice-product-search-export-products`
+    - console commands:
+        - `shopsys:elasticsearch:create-indexes` -> `shopsys:microservice:product-search:create-structure`
+        - `shopsys:elasticsearch:delete-indexes` -> `shopsys:microservice:product-search:delete-structure`
+        - `shopsys:elasticsearch:export-products` -> `shopsys:microservice:product-search:export-products`
+- run `php phing ecs-fix` to apply new coding standards
```

Comment:
I would appreciate the description of the new coding-standards, are we speaking about those yaml indents?

Reply:
no, improved